packages/tools/stylelint-config-pie/package.json

```diff
@@ -17,17 +18,17 @@
     "./ordering": "./rules/ordering.js"
   },
   "peerDependencies": {
-    "stylelint": "14.x",
-    "stylelint-config-standard-scss": "5.x",
-    "stylelint-order": "5.x"
+    "stylelint": "16.x",
+    "stylelint-config-standard-scss": "13.x",
+    "stylelint-order": "6.x"
   },
   "dependencies": {
     "@justeattakeaway/stylelint-no-logical-props-shorthands": "0.2.0"
   },
   "scripts": {
     "lint:scripts": "run -T eslint .",
     "lint:scripts:fix": "run -T eslint . --fix",
-    "test": "run -T vitest run --config ../../../vite.config.js",
+    "test": "run -T vitest run --config ../../../vite.config.js --environment node",
```

Comment:
adding environment node here as stylelint is expected to run in a node environment. 
more details are [here](https://stylelint.io/migration-guide/to-16)